templates/default.yaml

```diff
@@ -14,5 +14,13 @@ zephyr_repository_step:
   0. Go to [Anjay-zephyr-client](https://github.com/AVSystem/Anjay-zephyr-client/releases).
 nRF9160DK_binary_step:
   0. Download the `demo_nrf9160dk_merged.hex` file.
+nRF9160DK_flashing_nRF_docs_step:
+  0. To program the board, go through the whole process of **flashing nRF** [described here](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_nrf9160_gs.html).
+nRF9160DK_flash_downloaded_binary_step:
+  0. Use the nRF Connect Programmer with the downloaded `.hex` file and execute steps from the [Updating the application firmware](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_nrf9160_gs.html#updating-the-application-firmware) section.
```

Comment:
From what I can see these links are outdated. Maybe we can replace these two steps with something similar to `nRF7002` (not necessarily in this MR)?

Reply:
Could you copy here up to date links? I don't recognize current ones to be outdated, so this would be hard to me

Reply:
What I mean is that these links work but redirect to new page (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/device_guides/working_with_nrf/nrf91/nrf9160_gs.html) which has been redesign and there is no `updating-the-application-firmware` and `flashing-nrf` section. So we should update the links or replaced these two section. I think that https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_nrf9160_gs.html#updating-the-application-firmware can be replace with https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/device_guides/working_with_nrf/nrf91/nrf9160.html#id28 but probably we should write that user should use binaries downloaded in previous step instead of `nrf9160dk_asset_tracker_v2...` in step 7. But I'm not sure what we should replace the link from `nRF9160DK_flashing_nRF_docs_step:` section with. And as I wrote, it is not directly related to this task, so we can do it in a separate task.